docs/source/secrets.rst

```diff
@@ -1,33 +0,0 @@
-Secrets
```

Comment:
@mime-frame Just pushed a commit to also remove the link from the TOC in `index.rst`